packages/core/src/App/Containers/LinkExpiredModal/link-expired-modal.tsx

```diff
@@ -0,0 +1,114 @@
+import classNames from 'classnames';
+import { useFormik } from 'formik';
+import React, { useEffect } from 'react';
+import { Button, Dialog, Icon, Input, Text } from '@deriv/components';
+import { useVerifyEmail } from '@deriv/hooks';
+import { getErrorMessages, validEmail } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
+
+const LinkExpiredModal = observer(() => {
+    const { ui } = useStore();
+    const { enableApp, disableApp, is_link_expired_modal_visible: is_visible, toggleLinkExpiredModal } = ui;
+
+    const { send, error, data } = useVerifyEmail('reset_password');
+    const [is_email_sent, setIsEmailSent] = React.useState(false);
+
+    const formik = useFormik({
+        initialValues: { email: '' },
+        initialStatus: { error_msg: '' },
+        validate: values => (!validEmail(values.email) ? { email: getErrorMessages().email() } : {}),
+        onSubmit: values => send(values.email),
+    });
+
+    useEffect(() => {
+        if (data) {
+            formik.setSubmitting(false);
+            setIsEmailSent(true);
+        }
+    }, [data]);
+
+    useEffect(() => {
+        if (error && typeof error === 'object' && 'message' in error) {
```

Comment:
can we just check if (error?.message) ?

Reply:
the hook returns an error object which doesn't have any type. is there any way of suppressing this?
![Screenshot 2023-07-26 at 1 06 46 PM](https://github.com/binary-com/deriv-app/assets/122449658/7418d25a-ea05-45de-8382-f6c0ce3f13a9)